creator-node/test/snapbackSM.test.js

```diff
@@ -810,4 +811,274 @@ describe('test SnapbackSM', function () {
     assert.ok(requiredUpdateReplicaSetOps[0].unhealthyReplicas.has('http://deregisteredCN.co'))
     assert.strictEqual(potentialSyncRequests[0].endpoint, 'http://cnWithSpId2.co')
   })
+
+  it('[aggregateReconfigAndPotentialSyncOps] if the self node (primary) and 1 secondary are healthy but not the other secondary, issue reconfig for the unhealthy secondary', async function () {
+    const snapback = new SnapbackSM(nodeConfig, getLibsMock())
+
+    snapback.peerSetManager.endpointToSPIdMap = {
+      'http://some_healthy_primary.co': 1,
+      'http://cnWithSpId2.co': 2,
+      'http://unhealthyCnWithSpId3.co': 3
+    }
+
+    snapback.endpoint = 'http://some_healthy_primary.co'
+
+    const nodeUsers = [{
+      'user_id': 1,
+      'wallet': '0x00fc5bff87afb1f15a02e82c3f671cf5c9ad9e6d',
+      'primary': 'http://some_healthy_primary.co',
+      'secondary1': 'http://cnWithSpId2.co',
+      'secondary2': 'http://unhealthyCnWithSpId3.co',
+      'primarySpID': 1,
+      'secondary1SpID': 2,
+      'secondary2SpID': 3
+    }]
+
+    const unhealthyPeers = new Set(['http://unhealthyCnWithSpId3.co'])
+
+    const userSecondarySyncMetricsMap = await snapback.computeUserSecondarySyncSuccessRatesMap(nodeUsers)
```

Comment:
tracing through the code, it looks like this will return empty objects, prob should just explicitly mock it like you do below?